sklearn/decomposition/tests/test_incremental_pca.py

```diff
@@ -273,3 +273,17 @@ def test_whitening():
         assert_almost_equal(X, Xinv_ipca, decimal=prec)
         assert_almost_equal(X, Xinv_pca, decimal=prec)
         assert_almost_equal(Xinv_pca, Xinv_ipca, decimal=prec)
+
+
+def test_partial_fit_correct_answer():
```

Comment:
I'd rather have something like https://github.com/scikit-learn/scikit-learn/pull/9492#issuecomment-321190011 as a regression test. To me this seems less magical than comparing to a "known answer" whose provenance is not very clear.